hq/public/javascripts/app.js

```diff
@@ -36,17 +36,34 @@ $(document).ready(function() {
     }
   );
 
+  // filtering table results
```

Comment:
We can use [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData) here to simplify this a little. The benefit of `FormData` is we can won't need to repeat this line `$('.js-finding-filter-for-X').change(function() {` for each form, thus keeping things a DRY.

Here's an example https://jsfiddle.net/ahyr1wpc/.

Reply:
Thank you so much for detailing this here! I'm going to write up a ticket for this as it would be good to improve this across all the checkboxes used in SHQ.

Reply:
https://github.com/guardian/security-hq/pull/244